wear/src/main/java/io/homeassistant/companion/android/complications/views/ComplicationConfigMainView.kt

```diff
@@ -0,0 +1,135 @@
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.Button
+import androidx.wear.compose.material.ButtonDefaults
+import androidx.wear.compose.material.Chip
+import androidx.wear.compose.material.ChipDefaults
+import androidx.wear.compose.material.ExperimentalWearMaterialApi
+import androidx.wear.compose.material.Text
+import androidx.wear.compose.navigation.SwipeDismissableNavHost
+import androidx.wear.compose.navigation.composable
+import androidx.wear.compose.navigation.rememberSwipeDismissableNavController
+import com.mikepenz.iconics.compose.Image
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+import io.homeassistant.companion.android.HomeAssistantApplication
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.complications.ComplicationConfigViewModel
+import io.homeassistant.companion.android.data.SimplifiedEntity
+import io.homeassistant.companion.android.home.views.ChooseEntityView
+import io.homeassistant.companion.android.home.views.ListHeader
+import io.homeassistant.companion.android.home.views.ThemeLazyColumn
+import io.homeassistant.companion.android.theme.WearAppTheme
+import io.homeassistant.companion.android.theme.wearColorPalette
+import io.homeassistant.companion.android.util.getIcon
+
+private const val SCREEN_MAIN = "main"
+private const val SCREEN_CHOOSE_ENTITY = "choose_entity"
+
+@OptIn(ExperimentalWearMaterialApi::class)
+@Composable
+fun LoadConfigView(
+    complicationConfigViewModel: ComplicationConfigViewModel,
+    onAcceptClicked: () -> Unit
+) {
+    WearAppTheme {
+        val swipeDismissableNavController = rememberSwipeDismissableNavController()
+        SwipeDismissableNavHost(
+            navController = swipeDismissableNavController,
+            startDestination = SCREEN_MAIN
+        ) {
+            composable(SCREEN_MAIN) {
+                MainConfigView(
+                    entity = complicationConfigViewModel.selectedEntity.value,
+                    hasSelected = complicationConfigViewModel.hasSelected.value,
+                    loaded = complicationConfigViewModel.loadingState.value == ComplicationConfigViewModel.LoadingState.READY,
+                    error = complicationConfigViewModel.loadingState.value == ComplicationConfigViewModel.LoadingState.ERROR,
+                    onChooseEntityClicked = {
+                        swipeDismissableNavController.navigate(SCREEN_CHOOSE_ENTITY)
+                    },
+                    onAcceptClicked = onAcceptClicked
+                )
+            }
+            composable(SCREEN_CHOOSE_ENTITY) {
```

Comment:
The title for this screen is currently "Choose shortcuts", but should be "Choose entity" to match the chip that was just tapped